packages/core/saved-objects/core-saved-objects-migration-server-internal/src/actions/clone_index.ts

```diff
@@ -80,11 +88,16 @@ export const cloneIndex = ({
             refresh_interval: '1s',
             // Bump priority so that recovery happens before newer indices
             priority: 10,
-            // Increase the fields limit beyond the default of 1000
-            mapping: {
-              total_fields: { limit: 1500 },
-            },
-          },
+          }),
+    };
+
+    return client.indices
+      .clone({
+        index: source,
+        target,
+        wait_for_active_shards: WAIT_FOR_ALL_SHARDS_TO_BE_ACTIVE,
```

Comment:
not an index setting so it's not included in the latest stateless-es PR, but I found other docs stating that this option will be removed from several APIs. But given it's not blocked yet as far as I know, we can do that in a follow-up.